matlab/CMakeLists.txt

```diff
@@ -542,6 +542,23 @@ if(UNIX
   endif()
 endif()
 
+if(WIN32)
```

Comment:
It seems that we have similar code for `Apple`.
Can we unify them by using `CMAKE_SHARED_LIBRARY_PATH` variable?
https://cmake.org/cmake/help/latest/variable/CMAKE_SHARED_LIBRARY_SUFFIX.html

Reply:
Thank you for your suggestion. It would be great to unify the code. I'll try and get update soon.